src/include/rbd/librbd.h

```diff
@@ -181,6 +181,22 @@ enum {
   RBD_IMAGE_OPTION_DATA_POOL = 10
 };
 
+typedef enum {
+  RBD_DELAY_CONDITION = 0,
+  RBD_CLONES_DELETED_CONDITION = 1
+} rbd_remove_condition_type_t;
+
+typedef struct {
+  uint64_t delay;
+} rbd_delay_condition_t;
+
+typedef struct {
+  rbd_remove_condition_type_t type;
+  union {
+    rbd_delay_condition_t delay_cond;
+  } cond;
+} rbd_remove_condition_t;
```

Comment:
forget it. I misinterpreted the "date of deletion" as the date at which the image was purged from trash.